src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -1,236 +1,3 @@
-.ez-table {
```

Comment:
This wasn't deleted before because we still have (AFAIK) `ez-table--list`, which were not redesigned during table redesign.